internal/wallet/test_util.go

```diff
@@ -48,14 +48,14 @@ func setupDB(t *testing.T) (db walletdb.DB, tearDownFunc func()) {
 	return
 }
 
-func setupWallet(t *testing.T) (tearDownFunc func(), w Wallet) {
+func setupWallet(t *testing.T) (tearDownFunc func(), w Wallet, db walletdb.DB) {
```

Comment:
Some tests need to access the wallet's db directly, so that's why I added db as one of returned values